src/server.c

```diff
@@ -5326,6 +5391,38 @@ sds genRedisInfoString(const char *section) {
         raxStop(&ri);
     }
 
+    if (allsections || !strcasecmp(section,"latencystats")) {
+        /* Latency by percentile distribution per command category */
+        info = sdscatprintf(info, "# Latencystats - latency by percentile distribution\r\n");
```

Comment:
We basically never add information after the headname. 
```suggestion
        info = sdscatprintf(info, "# Latencystats\r\n");
```

Reply:
This is still here.

Reply:
@filipecosta90 you removed one, and kept the other, please remove the other header print completely (including `sections++` etc)